include/random.h

```diff
@@ -1,4 +1,4 @@
-/*
+/*! \page License
    KQ is Copyright (C) 2002 by Josh Bolduc
```

Comment:
Actually I think we only want one of these with the Doxygen markup. It'll create a standalone page called License in the generated documents, so there only needs to be one in the whole project, not one per file (maybe Doxygen is smart enough to merge them, I haven't checked)

Reply:
That would be great!

Reply:
I checked, on Doxygen 1.9.3 at least, it seems that if you have identical \\page identifiers with identical content you'll get one page with that content. If the content differs it will concatenate one of each of all the different versions. So, smarter than I gave it credit for!

Reply:
What are your thoughts? Remove from all files except maybe `kq.cpp` (since it includes `int main()`)? Keep them all in for now (maybe remove them in a separate PR)? Remove from everything and just write a LICENSE.md file?

Which, by the way, we probably need anyway if this is going to be using `gettext`, `tinyxml2` and others: we may need to include all those licenses somewhere anyway so it will be valid on the various Linux distributions.

Reply:
I'd leave them, but just remove the \page directive from all but one (on kq.cpp.) I thought for a moment we didn't actually include the licence anywhere but (as the header says!) it's in the file COPYING. I don't know what we do about our dependency libraries - it is probably enough to reference them at the end of the README file (just say KQ depends on XXX library copyright YYY and give the project URL)